src/lib/util/util.ts

```diff
@@ -346,6 +346,24 @@ export function getContent(message: Message): string | null {
 	return null;
 }
 
+/**
+ * Gets all the contents from a message.
+ * @param message The Message instance to get all contents from
+ */
+export function getAllContent(message: Message): string {
+	const output: string[] = [];
+	if (message.content) output.push(message.content);
```

Comment:
Don't we want cleanContent here? Or call cleanMentions otherwise?

Reply:
No, this will be used for moderation monitors and markov, they never echo to a message content.

Reply:
kk